.github/workflows/test.yml

```diff
@@ -205,3 +205,20 @@ jobs:
           path: basic
       - name: Verify basic
         run: __test__/verify-basic.sh --archive
+    
+  test-git-container:
+    runs-on: ubuntu-latest
+    container: bitnami/git:latest
```

Comment:
we will have e2e coverage on any newer git version release.

Reply:
มันน่าจะดีมากขึ้นไปอีก ผมเคารพการตัดสินใจของชุมชน